.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: ruby
-rvm: 2.7.4
+rvm: 3.3.3 # Pinned it to this version because the current stable
+           # version 3.3.4 isn't available in 'travis-ci-language-archive' yet
```

Comment:
Can you use either a block comment before the relevant code line, or a single in-line comment? No hybrids like this, please.

Reply:
Sure Adrian, Removed this in https://github.com/apel/apel.github.io/pull/27/commits/e42af99e0486edc2bfe8381ca5daa56b604637f6.